workerapi/server/server.go

```diff
@@ -0,0 +1,102 @@
+package server
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/luci/go-render/render"
+	"github.com/scootdev/scoot/workerapi/gen-go/worker"
+)
+
+func Serve(handler worker.Worker, addr string, transportFactory thrift.TTransportFactory, protocolFactory thrift.TProtocolFactory) error {
+	transport, err := thrift.NewTServerSocket(addr)
+	if err != nil {
+		return err
+	}
+	processor := worker.NewWorkerProcessor(handler)
+	server := thrift.NewTSimpleServer4(processor, transport, transportFactory, protocolFactory)
+
+	fmt.Println("About to serve")
+
+	return server.Serve()
+}
+
+type Handler struct {
+	runs map[string]*worker.RunStatus
```

Comment:
Would it be easier to use the no-op runner that currently exists to do this?

(That would make this file just a translator between the Thrift objects and the domain objects, and then call the Runner for the business logic)